symphony/bdk/core/service/message/message_service.py

```diff
@@ -135,25 +167,54 @@ async def list_messages(
 
         """
         params = {
-            'sid': stream_id,
-            'since': since,
-            'session_token': await self._auth_session.session_token,
-            'key_manager_token': await self._auth_session.key_manager_token,
-            'skip': skip,
-            'limit': limit
+            "sid": stream_id,
+            "since": since,
+            "session_token": await self._auth_session.session_token,
+            "key_manager_token": await self._auth_session.key_manager_token,
+            "skip": skip,
+            "limit": limit
         }
         message_list = await self._messages_api.v4_stream_sid_message_get(**params)
         return message_list.value
 
-    @retry
     async def blast_message(
```

Comment:
and if we want to push it one step further we could do the blast with the send message, allowing a list of stream ids
but given it is a different API call I would keep it a different service method